jsonnet/telemeter/prometheus/kubernetes/kubernetes.libsonnet

```diff
@@ -0,0 +1,339 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+
+{
+  _config+:: {
+    versions+:: {
+      openshiftOauthProxy: 'v1.1.0',
+      prometheus: 'v2.3.2',
+    },
+
+    imageRepos+:: {
+      openshiftOauthProxy: 'openshift/oauth-proxy',
+      prometheus: 'quay.io/prometheus/prometheus',
+    },
+
+    prometheus+:: {
+      namespace: 'telemeter',
```

Comment:
cc @metalmatze from our discussion of Friday.

All of the prometheus-telemeter resources need to end up in a different namespace than the telemeter-client and other resources generated by jsonnet, thus the specific declaration here. I added an `overrideNamespace` function to `main.jsonnet` that would modify the namespace of rendered resources, but after implementing it I realized that the namespace can be used in more places than just `metadata.namespace`: it is also used in rolebindings and in servicemonitors, so the `overrideNamespace` function was not sufficient and I reverted it.

What are your thoughts?

Reply:
Probably generating this from a completely new entrypoint is your only real option now. :confused: Sorry.

Reply:
:tada: done